Library/Formula/libsearpc.rb

```diff
@@ -0,0 +1,64 @@
+class Libsearpc < Formula
+  homepage "http://www.seafile.com/"
+  url "https://github.com/haiwen/libsearpc/archive/v3.0-latest.tar.gz"
+  sha256 "56313771e0ad7dc075c4590b6a75daeb3939937b21716d82c91be2612133b8cd"
```

Comment:
Could you please add `version "3.0"`? Homebrew tries to guess the version from the URL and fails (it assumes it’ `latest`).